packages/earth-admin/src/components/organizations/organization-edit/OrganizationEdit.tsx

```diff
@@ -65,27 +77,30 @@ export default function OrganizationEdit(props: OrganizationEditProps) {
                 Organization name
               </label>
               <input
-                ref={register}
+                ref={register({
+                  required: true,
```

Comment:
Does this required flag affect the edit organization flow? Or is it no longer required if the field is diabled?

Reply:
In "edit mode", the value will be there already - it's always required, to make it more clear.